### PR TITLE
etherlime init and etherlime deploy commands

### PR DESCRIPTION
**Pull request by Perseverance**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +20,6 @@ This framework was born out of necessity, hardships and trouble in the developme
 ```
 npm install etherlime
```
Comment (thcrnk) [inline README.md]: Do we need -g here?

Comment (Perseverance) [reply README.md]: No not really, it is developers choice how to use it. Both will work

Comment (thcrnk) [reply README.md]: If installed without -g the run command etherlime deploy returns following - bash: etherlime: command not found

Comment (Perseverance) [reply README.md]: much like truffle. you'd need to run it from `node_modules/.bin/etherlime`

Comment (Perseverance) [reply README.md]: I'll change it to `npm install etherlime [-g]`

**Pull request merged**